### PR TITLE
Remove LDVerifierKeyPair and export helper functions.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
* Remove `LDVerifierKeyPair` subclass. Fold `signer()` and `verifier()` methods into parent `LDKeyPair` class.
* Remove `keyPair.addPrivateKey()` and `keyPair.addPublicKey()`. Subclasses will have to override `export()` directly.

(per @dlongley's request).

### CODE REVIEW

File: lib/LDKeyPair.js [hunk 1]
```diff
@@ -138,6 +115,52 @@ class LDKeyPair {
...
+   * > signer
+   * { sign: [AsyncFunction: sign] }
+   * > signer.sign({data});
+   *
+   * @returns {{sign: Function}} A signer for json-ld usage.
+   */
+  /* eslint-enable */
+  signer() {
+    return {
+      async sign({/* data */}) {
```
Comment (dlongley) [inline lib/LDKeyPair.js:136]: I can't recall -- but does the signer API also need an `id` property? I feel as though there's an `id` property used to populate things like `verificationMethod` when using a signer.

**Review by dlongley — APPROVED** [review]
+1 to reducing extra classes when they aren't needed.

**Review by mattcollier — APPROVED** [review]
There's an unresolved comment from Longley.  Otherwise, LGTM, TY.

File: lib/LDKeyPair.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/LDKeyPair.js:136]: @dlongley - you're thinking of the higher-level jsigs.sign() functionality (that uses the key id for verificationMethod). This `signer()` is much more lower level, and uses the built-in node crypto to sign a buffer, so it doesn't use an id.

Comment (dmitrizagidulin) [reply lib/LDKeyPair.js:136]: https://github.com/digitalbazaar/ed25519-verification-key-2020/blob/main/lib/Ed25519VerificationKey2020.js#L212

**Pull request merged**